jq.c

```diff
@@ -1300,12 +1348,43 @@ jq_get_exception()
 		{
 			ereport(ERROR, (errmsg("java/lang/Object class could not be created")));
 		}
+		// Stack Trace
+		{
+			jobject sw = NewStringWriter(Jenv);
+			if (sw == NULL) {
+				ereport(ERROR, (errmsg("String Writer is null")));
```

Comment:
Because our purpose is to print stack trace at debug level (DEBUG3), so I think this error should only be displayed if DEBUG3 level is set in postgresql.conf.